sig-k8s-infra/charter.md

```diff
@@ -0,0 +1,186 @@
+# SIG K8s Infra Charter
+
+This charter adheres to the conventions described in the
+[Kubernetes Charter README] and uses the Roles and Organization Management
+outlined in [sig-governance].
+
+## Scope
+
+The successful migration of ownership and management of all Kubernetes
+project infrastructure from the google.com GCP Organization 
+(or elsewhere) to the CNCF, such that the Kubernetes project is able to
+sustainably operate itself without direct assistance from external vendors
+or entities.
+
+In other words, we seek to eradicate usage of the phrase "oh that's
+something that only an employee of Vendor X can do, we're blocked until
+they respond."
+
+### In scope
+
+#### Code, Binaries and Services
+
+Code, data and policies necessary to provision, update, decommission and
+otherwise manage all project infrastructure. Given that this is a broad scope,
+we prefer (where possible) to delegate ownership and operation of the 
+code / infrastructure to more directly responsible SIGs or Committees. This
+is largely how the SIG operated during its lifetime as a WG, driving the
+policies and tooling upon which SIG-owned infrastructure operates.
+
+Areas of responsibility include:
+
+- Policy definition and enforcement for areas related to project
+  infrastructure, including:
+  - What is in-scope/out-of-scope for project infrastructure
+  - Who should be allowed access to which parts of project infrastructure,
+    e.g. team definition, vetting criteria, etc.
+  - How infrastructure should be managed, e.g. naming schemes, acceptable
+    tooling or practices, on-call or escalation policies, etc.
+- Configuration management of all resources and service usage within the
+  kubernetes.io GCP Organization, including, but not limited to:
+  - API / Service enablement
+  - BigQuery datasets
+  - DNS records, e.g. for k8s.io, kubernetes.io, and other project-owned domains
+  - GCB usage
+  - GCP projects, instances, images
+  - GCR repositories
+  - GCS buckets
+  - GKE clusters, e.g. community infra cluster, prow build clusters
+  - GSM secrets
+  - Google Groups
```

Comment:
This one seems like a shared one with Steering/ContribEx

Reply:
I thought we covered this under the `Public Names` section?  I have no intention of kicking anyone out of OWNERS files so as to ensure policy is enforced.  Ownership of the code I'm gonna push back and suggest that it makes more sense for this SIG to own it since the rest of our infrastructure breaks if this code breaks.